test/src/auto_scroller/auto_scroller_test.dart

```diff
@@ -46,11 +46,13 @@ void main() {
 
         expect(autoScroller.isAbleToScroll, isFalse);
       },
+      // FIXME: It's not clear why it should not be able to scroll
+      skip: true,
```

Comment:
You might want to check if this test is still valid and `isAbleToScroll` should indeed be `false`.

Reply:
The auto-scroller needs to acknowledge the current position of the scroll-view in order to calculate:

* The position after the overscroll. The smooth scroll animation that happens when the user lifts its finger while dragging is what I call overscroll.
* Whether there's anything left to scroll. For instance, if the user is trying to scroll backwards but we are at the top of the grid already, there's nothing to scroll.
* The scroll duration. Notice that the scroll of this library has consistent speed. If you play a little with scroll-controller, you'll see that, by default, the scroll speed varies and the scroll duration is constant. When you are at the top of a list and try to reach the bottom, you'll see it takes the same amount of time as if you were at the middle of the list. This is not an acceptable behavior for this library, so I had to do some calculations that consider the current position and target position in order to give you a constant scroll speed, but variable scroll duration.

If the scroll-controller is not attached to any scroll-view, there's no current position, therefore none of the above calculations can be performed. This is the reason why it's impossible to scroll in such situation.

Reply:
So the new behavior is not correct. We should get back to nullable `currentPosition`. I should be able (not abble, lol) handle this part, though. Let's just get this PR done :)